pkg/features/kube_features.go

```diff
@@ -572,6 +572,13 @@ const (
 	// Enables SleepAction in container lifecycle hooks
 	PodLifecycleSleepAction featuregate.Feature = "PodLifecycleSleepAction"
 
+	// owner: @sreeram-venkitesh
+	// kep: http://kep.k8s.io/4818
+	// alpha: v1.32
```

Comment:
We may remove this line as https://github.com/kubernetes/kubernetes/pull/128076.